packages/core/src/components/deprecated-text/LargeText.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import { Text, TextProps } from "../text/Text";
+
+/**
+ * @deprecated Please use `Text` instead.
+ */
+export const LargeText: React.FC<Omit<TextProps, "variant">> = (props) => {
```

Comment:
Should this be `"size"`? Since we are setting `size` on the component?

Reply:
Oh, nice catch! Yes, you are correct!